tests/system/python/conftest.py

```diff
@@ -672,6 +637,8 @@ def pytest_addoption(parser):
                      help="PI Server user login password")
     parser.addoption("--pi-token", action="store", default="omf_north_0001",
                      help="OMF Producer Token")
+    parser.addoption("--pi-use-legacy", action="store", default="true",
+                     help="Set true if you want to send linked data types otherwise set true.")
```

Comment:
Set false if you want to send linked data types otherwise set true.

Reply:
Set false to override the default plugin behaviour i.e. for OMF version >=1.2.x to send linked data types